docs/binary.md

```diff
@@ -24,7 +29,9 @@ Binary Number :: 11001
 
 ---
 
-### How do we count using binary?
+## Binary Counting
+
+<h2>How do we count using binary?</h2>
```

Comment:
Any particular reasons for using a heading tags in markdown?

Reply:
Also, please follow the sentence case guidelines in headings mentioned [here](https://developers.google.com/style/capitalization)

Reply:
Thanks @Dishebh